Package.swift

```diff
@@ -177,10 +177,7 @@ let package = Package(
         ),
         .target(
             name: "SmithyHTTPAPI",
-            dependencies: [
-                "Smithy",
-                .product(name: "AwsCommonRuntimeKit", package: "aws-crt-swift")
-            ]
+            dependencies: ["Smithy"]
```

Comment:
CRT dependency is removed from `SmithyHTTPAPI` since no remaining types in `SmithyHTTPAPI` reference it.